### PR TITLE
Add subpackge to enable ptrace

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: aaa_base.spec [hunk 1]
```diff
@@ -103,6 +103,18 @@ This package includes some special settings needed on Windows Subsystem
...
 
+%package yama-enable-ptrace
+Summary:        sysctl setting to allow ptrace with the YAMA LSM enabled
+Group:          System/Fhs
+Requires:       %{name} = %{version}
+
+%description yama-enable-ptrace
+When the YAMA LSM is enabled, ptrace is restriced by default. On
+developer systems this has an impact on e.g. strace and gdb. So
+this package contains a settint that allows ptrace again.
```
Comment (thkukuk) [inline aaa_base.spec:114]: /settint/setting/

**Comment by johseg** [comment]
I think that this is a good idea. Makes it easier for users with this need to get the old behaviour back

**Comment by giulianobelinassi** [comment]
So this should be as easy as adding
```
BuildRequires: yama-enable-ptrace
```
on the .spec file, right? If yes then it is fine to me.

**Comment by giulianobelinassi** [comment]
ping.

This is needed for getting libpulp back in Tumbleweed.

**Review by thkukuk — APPROVED** [review]

**Pull request merged**